### PR TITLE
[Testing] [Notebook] fix failing notebook tests from three-dot-menu-proto

### PR DESCRIPTION
**Pull request by nikhilmandlik**
### Resolves: 
resolves: #3463 [Testing] [Notebook] fix failing notebook tests from three-dot-menu-proto 

### Author Checklist:
Check | Passed
-- | --
Changes address original issue? | Y
Unit tests included and/or updated with changes? | Y
Command line build passes? | Y
Changes have been smoke-tested? | Y

### CODE REVIEW

File: src/plugins/notebook/pluginSpec.js [hunk 1]
```diff
@@ -133,4 +133,94 @@ describe("Notebook plugin:", () => {
...
         });
     });
+
+    describe("Notebook Snapshots view:", () => {
+        let snapshotIndicator;
+        let drawerElement;
+
+        function clickSnapshotIndicator() {
+            const indicator = element.querySelector('.icon-camera');
+            // console.log('clickSnapshotIndicator', indicator);
```
Comment (davetsay) [inline src/plugins/notebook/pluginSpec.js:143]: remove console log

File: src/plugins/notebook/pluginSpec.js [hunk 2]
```diff
@@ -133,4 +133,94 @@ describe("Notebook plugin:", () => {
...
+
+            clickSnapshotIndicator();
+            classes = drawerElement.classList;
+            const isExpandedAfterSecondClick = classes.contains('is-expanded');
+
+            const success = isExpandedBefore === false
+                && isExpandedAfterFirstClick === true
+                && isExpandedAfterSecondClick === false;
+
+            expect(success).toBe(true);
```
Comment (davetsay) [inline src/plugins/notebook/pluginSpec.js:214]: Up to you but to me this is clearer.
```suggestion
            expect(isExpandedBefore).toBeFalse();
            expect(isExpandedAfterFirstClick).toBeTrue();
            expect(isExpandedAfterSecondClick).toBeFalse();
```

File: src/plugins/notebook/pluginSpec.js [hunk 3]
```diff
@@ -133,4 +133,94 @@ describe("Notebook plugin:", () => {
...
+            const isExpandedBefore = classes.contains('is-expanded');
+
+            clickSnapshotIndicator();
+            classes = drawerElement.classList;
+            const isExpandedAfterFirstClick = classes.contains('is-expanded');
+
+            const success = isExpandedBefore === false
+                && isExpandedAfterFirstClick === true;
+
+            expect(success).toBe(true);
```
Comment (davetsay) [inline src/plugins/notebook/pluginSpec.js:195]: see next comment about explicit expects

**Review by davetsay — CHANGES_REQUESTED** [review]
Thanks for fixing tests. Made a couple minor comments. Also, this isn't in your changes, but it might be best to move the var definitions inside the describe block scope. I don't know exactly if it makes a difference in the way jasmine picks up test files, but at the very least for consistency with other Spec file format.

**Review by shefalijoshi — APPROVED** [review]
LGTM
### Reviewer Checklist

1. Changes appear to address issue? Y
2. Appropriate unit tests included? N
3. Code style and in-line documentation are appropriate? Y
4. Commit messages meet standards? Y
5. Has associated issue been labelled `unverified`? (only applicable if this PR closes the issue) Y

**Review by davetsay — APPROVED** [review]

**Pull request merged**